src/feature-flags.ts

```diff
@@ -0,0 +1,36 @@
+/** All registered GK togglers. Do not manually add items to it. */
+const GateKeeperTogglers: Record<string, () => void> = {};
+
+// Add your new feature flags below
+
+export const isRequirementDebuggerEnabled = registerFeatureFlagChecker('RequirementDebugger');
+
+// Add your new feature flags above
+// Gate keeper runtime code below. Do not touch if you do not understand.
+
+/**
+ * @param flagName name of the feature flag. Use `PascalCase` to be callable in developer console.
+ * @returns a checker function to tell whether the feature flag is enabled.
+ */
+function registerFeatureFlagChecker(flagName: string): () => boolean {
```

Comment:
Instead of `flagName: string` can we define a more specific type? (similar to how we do gtags)

For example:
```
type FlagType =
  | 'RequirementDebugger'; // shows or hides the Requirement Debugger button
...
function registerFeatureFlagChecker(flagName: FlagType): () => boolean { ...
```